pallets/delegation/src/lib.rs

```diff
@@ -485,7 +485,8 @@ impl<T: Config> Pallet<T> {
 		Ok((revocations, consumed_weight))
 	}
 
-	// Revoke all children of a delegation
+	/// revoke_children revokes all children of a delegation.
```

Comment:
Who not
```suggestion
	/// Revokes all children of a delegation.
```

Reply:
Mh Maybe it was a go conversion to always have the method name in the beginning of the doc...